linkerd/drain/src/lib.rs

```diff
@@ -114,136 +116,147 @@ impl Watch {
 
 impl ReleaseShutdown {
     /// Releases shutdown after `future` completes.
+    #[inline]
```

Comment:
i'm not sure if this is meaningful on `async fn`s...i don't think they can get inlined? might be worth asking someone who knows about rustc.